fedora/client/openidbaseclient.py

```diff
@@ -182,7 +183,8 @@ def _authed_get(self, url, params=None, data=None, **kwargs):
         response = self._session.get(url, params=params, data=data, **kwargs)
         return response
 
-    def send_request(self, method, auth=False, verb='POST', **kwargs):
+    def send_request(self, method, auth=False, verb='POST', retries=None,
+                     retry_sleep=0.5, **kwargs):
```

Comment:
Maybe I'm mistaken, but how can I currently configure `retry_sleep`? If I do `bodhi = Bodhi2Client(); bodhi.retry_sleep = 2`, it seems it won't have any effect in `send_request()` (because you don't utilize the class attribute).

Possible solutions that I see:
1. Remove `retry_sleep` from `send_request()` parameters. Use `self.retry_sleep` inside the method body.
2. Use `send_request(retry_sleep=None)` as a default value, and then have `retry_sleep = retry_sleep if retry_sleep is not None or self.retry_sleep` (or use multiple lines if you don't like ternary operator) inside the method body.